packages/@aws-cdk/aws-sns-subscriptions/lib/subscription.ts

```diff
@@ -10,4 +11,12 @@ export interface SubscriptionProps {
    * @default - all messages are delivered
    */
   readonly filterPolicy?: { [attribute: string]: sns.SubscriptionFilter };
+
+  /**
+   * Queue used when `deadLetterQueue` is enabled.
+   * Default queue is created when not specified.
+   *
+   * @default - SQS queue with 14 day retention period if `deadLetterQueueEnabled` is `true`
```

Comment:
```suggestion
   * Queue for messages that fail to reach the subscription destination a certain number of times.
   *
   * @default - No dead letter queue enabled.
```

Reply:
@SeekerWing The top of these doc blocks still refer to `deadLetterQueueEnabled`.